cypress/e2e/contactForm.cy.js

```diff
@@ -1,16 +1,16 @@
 import ContactFormPageObject from '../support/pages/contactForm.pageObject';
 import HomeAndCataloguePageObject
```

Comment:
There is a missing semicolon at the end of this import statement. This might cause issues depending on your project's configuration.